src/bin/ipa_bench/net/client/mod.rs

```diff
@@ -6,6 +6,7 @@ use thiserror::Error as ThisError;
 
 use super::Command;
 
+#[allow(dead_code)]
```

Comment:
it is not ideal to sprinkle these suppressions everywhere, in my experience it usually happens when I forget to declare module public. is it the same here?

Reply:
@taikiy would you intend to use this code in MpcHandle and MpcHttpConnection or is it safe to purge that code for now?